pom.xml

```diff
@@ -442,12 +447,13 @@
         </plugins>
     </build>
     <properties>
-        <carbon.kernel.version>4.4.32</carbon.kernel.version>
+        <carbon.kernel.version>4.6.0-alpha3</carbon.kernel.version>
```

Comment:
We need to update the relevant import versions of the kernel in all the components.

Reply:
Resolved in https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/53/commits/99c9587ba62eccd099e371f7cda4cec4aa8c649b